src/containers/tutor-profile/profile-info/ProfileInfo.jsx

```diff
@@ -6,27 +6,30 @@ import Button from '@mui/material/Button'
 import IconButton from '@mui/material/IconButton'
 import EditOutlinedIcon from '@mui/icons-material/EditOutlined'
 import CopyRoundedIcon from '@mui/icons-material/ContentCopyRounded'
+import StarIcon from '@mui/icons-material/Star'
+import Typography from '@mui/material/Typography'
 
 import ProfileContainerDesktop from '~/containers/tutor-profile/profile-info/ProfileContainerDesktop'
 import ProfileContainerMobile from '~/containers/tutor-profile/profile-info/ProfileContainerMobile'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import useBreakpoints from '~/hooks/use-breakpoints'
+import HashLink from '~/components/hash-link/HashLink'
 
 import { useSnackBarContext } from '~/context/snackbar-context'
-import {
-  accountInfoMock,
-  subjectChipsMock,
-  doneItemsMock
-} from '~/containers/tutor-profile/profile-info/ProfileInfo.constants'
 import { styles } from '~/containers/tutor-profile/profile-info/ProfileInfo.styles'
 import { snackbarVariants, myProfilePath } from '~/constants'
 import { SizeEnum } from '~/types'
+import { getNumberOfYears } from '~/utils/helper-functions'
 
-const ProfileInfo = () => {
+const ProfileInfo = ({ userData }) => {
   const { t } = useTranslation()
   const { isDesktop, isMobile } = useBreakpoints()
   const { setAlert } = useSnackBarContext()
   const isMyProfile = useMatch(myProfilePath)
+  const { number, format } = getNumberOfYears(
+    userData.createdAt,
+    userData.updatedAt
```

Comment:
You receive the wrong data, cause you need to pass `userData.createdAt` and real data.

Reply:
DOne